ra/ra-api-next.rng

```diff
@@ -7,6 +7,14 @@
 
 	<element name="version"> <text /> </element>
 
+	<zeroOrMore><element name="longdesc">
+		<ref name="description" />
+	</element></zeroOrMore>
+
+	<zeroOrMore><element name="shortdesc">
+		<ref name="description" />
+	</element></zeroOrMore>
```

Comment:
I just noticed we have oneOrMore for the parameter descriptions. I'm not sure why we would ever want more than one short/long description, whether for a parameter or here. I'd expect "optional" instead.

Reply:
I understand your concerns. The main reason why to have more description is to support multiple languages.

Reply:
Ah, that makes sense!

Reply:
How would one indicate which description belongs to which language?

Reply:
@krig Attribute 'lang', theoretically it is always required but it is missing quite often.